radis/io/hitemp.py

```diff
@@ -79,6 +46,35 @@
 # TODO: WIP. Maybe move somewhere else to be also used by HITRAN queries
 
 
+# Add a zip opener to the datasource _file_openers
```

Comment:
If it works well you could even push that to Numpy and improve their Datasource!